.ci/update-github-milestone.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+checkForVariable() {
+  VAR_NAME=$1
+  if [ ! -v "$VAR_NAME" ]; then
+    echo "Error: Define $1 environment variable"
+    exit 1
+  fi
+}
+
+checkForVariable "READ_ONLY_TOKEN"
+
+LATEST_RELEASE_TAG=$(curl -s https://api.github.com/repos/checkstyle/checkstyle/releases/latest \
+                       | jq ".tag_name" | sed 's/checkstyle-//' | sed 's/"//' | sed 's/"//')
+echo LATEST_RELEASE_TAG="$LATEST_RELEASE_TAG"
```

Comment:
remove, this is is not required.
You should take `title` value from Github action as parameter.

`.ci/update-github-milestone.sh $RELEASE_VERSION`

Milestone ID you take by `milestones?state=open` (already done by you), and you just set new title to such milestomne.

Reply:
Changes Done